src/schema/partner_artist.js

```diff
@@ -97,26 +96,10 @@ export const partnersForArtist = (artist_id, options, loader) => {
   }
 
   return loader(gravityArgs).then(({ body, headers }) => {
-    const connection = connectionFromArraySlice(body, options, {
+    return connectionFromArraySlice(body, options, {
       arrayLength: headers["x-total-count"],
       sliceStart: offset,
+      resolveNode: node => node.partner, // Can also be a promise: `partnerLoader(node.partner.id)`
```

Comment:
The `node` variable here is really the `edge` (`partner` is the node).

Reply:
Ah, yes.